apps/entropy-tester/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint"],
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  rules: {
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
+  },
+};
```

Comment:
A few notes:

1. `.eslintrc.js` is [a deprecated config format](https://eslint.org/docs/latest/use/configure/configuration-files-deprecated)
2. This is a pretty loose config that won't catch a ton of issues, including many issues that make the type system actually reliable and trustworthy.

The easiest fix is to replace this file with a file `eslint.config.js` that exports my canned config, e.g. [something like this](https://github.com/pyth-network/pyth-crosschain/blob/main/apps/entropy-explorer/eslint.config.js).  Note you'll need to set `"type": "module"` in your `package.json` to use this as I expect esm for all packages I support.